### DISCUSSION TITLE
Update avocado website

### DISCUSSION THREAD

**Discussion by ana** [avocado-framework/avocado#4981] [Show and tell]
I've worked in revamping avocado's website using jekyll - the static site generator preferred by GitHub pages. The biggest advantages are being able to write the website directly in markdown without having to mess with HTML and having an easy to modify theme template.

You can see the result at https://ana.github.io/avocado.github.io/ 
The code is available at https://github.com/ana/avocado.github.io
To run it locally you'll need to install jekyll and run `jekyll serve`.

I'm asking for early feedback because this is still work in progress.

**Comment by beraldoleal** [comment]
Hi @ana super cool. Thanks for taking care of this.

The only feedback that I have now, is that white font with the avocado background makes difficult to read the title. Also I think it is important to have a link to our root documentation, some sections are not linked there (i.e API reference, blueprints, etc..).

**Reply by pevogam** [reply]
I was about to say the same about the white font :+1:

**Comment by ana** [comment]
Website was updated, see https://github.com/avocado-framework/avocado/issues/5265
